practice-one/src/styles/sections/benefits.css

```diff
@@ -0,0 +1,84 @@
+.section-benefits {
+  display: flex;
+  gap: 56px 0;
+  flex-direction: column;
+  align-items: center;
+}
+
+.benefits-heading {
+  font-size: var(--font-size-2xl);
+  line-height: 50px;
+}
+
+.benefit-card {
+  display: flex;
+  align-items: center;
+  box-sizing: border-box; /* need to apply this style again */
+  width: 1060px;
+  height: 304px;
+  background-color: var(--background-color-card);
+
+  &:nth-of-type(1) {
+    padding: 25px 110px 25px 38px;
+    background-image: url("../../assets/images/working-hour.png");
+    background-repeat: no-repeat;
+    background-position: right 110px top 25px;
+  }
+
+  &:nth-of-type(2) {
+    flex-direction: row-reverse;
+    margin-left: 85px;
+    padding: 39px 161px 39px 110px;
+    background-image: url("../../assets/images/earning.png");
+    background-repeat: no-repeat;
```

Comment:
Add this style to the 'benefit-card' class instead of duplicating it for each item. Also, ensure that the background-position is the same for each item